kuma/javascript/src/article.jsx

```diff
@@ -36,6 +40,17 @@ const styles = {
         '& a.sectionLink': {
             fontSize: 24,
             textDecoration: 'none'
+        },
+
+        // Styles for BCD tables, overriding the stylesheets
+        '& table.bc-table button.bc-history-link': {
+            ':focus': {
+                outline: '#83d0f2 solid 3px',
+                outlineOffset: -3
```

Comment:
The outlining seems a bit off (I'm using the latest release of Firefox):
<img width="311" alt="image" src="https://user-images.githubusercontent.com/3743693/60219632-04e8f300-9828-11e9-9bc4-f6b8e779faca.png">

Reply:
As you told me in Slack @davidflanagan , it looks fine in Chrome:
<img width="196" alt="image" src="https://user-images.githubusercontent.com/3743693/60219851-fb13bf80-9828-11e9-8b8a-7dfcb9cf5281.png">

Reply:
@davidflanagan and I discussed this in Slack, and we're going to leave this for a future adjustment.